### PR TITLE
feat(workflow): Add toast to issue stream when taking an action on issue

### PR DESCRIPTION
**Pull request by mikellykels**
Add toast to the issue stream when taking an action on an issue (Resolve, Ignore, Mark Review, Merge, and Delete).

**Resolved**

_1 issue selected_
Screen Shot 2022-03-25 at 2 23 09 AM

_Multiple issues selected_
Screen Shot 2022-03-25 at 2 42 17 AM

**Ignored**

_1 issue selected_
Screen Shot 2022-03-25 at 2 23 20 AM

_Multiple issues selected_
Screen Shot 2022-03-25 at 3 15 12 AM

**Mark Reviewed**

_1 issue selected_
Screen Shot 2022-03-25 at 2 24 18 AM

_Multiple issues selected_
Screen Shot 2022-03-25 at 2 41 55 AM

**Merge**

Screen Shot 2022-03-25 at 2 45 44 AM

**Delete**

_1 issue selected_
Screen Shot 2022-03-25 at 2 41 29 AM

_Multiple issues selected_
Screen Shot 2022-03-25 at 2 41 38 AM

### CODE REVIEW

File: static/app/stores/groupStore.tsx [hunk 1]
```diff
@@ -358,14 +358,20 @@ const storeConfig: Reflux.StoreDefinition & Internals & GroupStoreInterface = {
   },
 
   onDeleteSuccess(_changeId, itemIds, _response) {
+    const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).toString();
```
Comment (evanpurkhiser) [inline static/app/stores/groupStore.tsx:361]: do we want `toString` or do we want
```suggestion
    const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).join(', ');
```

File: static/app/views/issueList/actions/index.tsx [hunk 2]
```diff
@@ -200,9 +203,7 @@ class IssueListActions extends React.Component<Props, State> {
           ...selection.datetime,
         },
         {
-          complete: () => {
-            clearIndicators();
-          },
+          complete: () => {},
```
Comment (evanpurkhiser) [inline static/app/views/issueList/actions/index.tsx:206]: does this have to be passed, can we just remove the property?

File: static/app/views/issueList/overview.tsx [hunk 3]
```diff
@@ -1073,7 +1080,16 @@ class IssueListOverview extends React.Component<Props, State> {
...
 
-  onIssueAction = (itemIds: string[]) => {
+  onIssueAction = (itemIds: string[], actionType: string) => {
+    if (itemIds.length > 1) {
+      addMessage(t(`${actionType} ${itemIds.length} Issues`), 'success', {
+        duration: 4000,
+      });
+    } else {
+      const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).toString();
+      addMessage(t(`${actionType} ${shortId}`), 'success', {duration: 4000});
```
Comment (evanpurkhiser) [inline static/app/views/issueList/overview.tsx:1090]: Why do we set a custom duration?

Comment (mikellykels) [reply static/app/views/issueList/overview.tsx:1090]: I set this to be the same as `showAlert` that the original delete/merge used
https://github.com/getsentry/sentry/blob/6b4dc91d5cf5d886a15cca7e05dc91d8613f2707/static/app/stores/groupStore.tsx#L17-L19

I'll also be adding an "Undo" button to the toast (except for delete/merge), and wanted to duration to be the same for all of them

**Review by scttcper — APPROVED** [review]
looks good

**Pull request merged**